### PR TITLE
#181339491-requester-table

### PR DESCRIPTION
**Pull request by Code-250**
### What does this PR do?
this PR implements requester dashboard stats with its mobile version

### Description of task
- create requester tables dashboard
- create requester stats dashboard
- implement requester CRUD operations
### How should this be tested Manually?

- `git checkout  ~/Dow/p/cabal-bn-fe  feature/#181339491-requests-table`
- `yarn install` to install all dependencies
- `yarn dev` to start the project
### ScreenShot
- web
Screen Shot 2022-05-24 at 17 13 32

- mobile view

Screen Shot 2022-05-24 at 17 22 14

### CODE REVIEW

**Review by Eli250 — CHANGES_REQUESTED** [review]
Your deployment is not showing anything when you click login.

**Review by Eli250 — APPROVED** [review]

**Review by Eli250 — APPROVED** [review]

**Review by Gabin-ishimwe — CHANGES_REQUESTED** [review]

**Review by Gabin-ishimwe — APPROVED** [review]

File: src/components/header.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+      sx={{
+        display: 'flex',
+        justifyContent: 'space-between',
+        alignItems: 'center',
+        width: '200px',
+      }}
+    >
+      <Typography variant="h6" sx={{ color: '#1A2D6D' }}>
+        Richard
+      </Typography>
```
Comment (ncutixavier) [inline src/components/header.js:30]: Do not hardcode anything, use data of logged in user

**Review by ncutixavier — CHANGES_REQUESTED** [review]

File: src/components/header.js (hunk 1 above)
Comment (Code-250) [reply src/components/header.js]: I have worked on the comment you gave me.

File: src/components/mobLinks.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+import { Link, Typography, Box, Container, Paper } from '@mui/material';
+import { matchPath } from 'react-router-dom';
+import React from 'react';
+
+const MobLink = ({ sideBarLinks }) => (
+  <Box sx={{ display: 'flex', justifyContent: 'space-around' }}>
+    {sideBarLinks.map((links) => (
+      <Paper elevation={0}>
+        <Link
+          href={links.to}
```
Comment (ncutixavier) [inline src/components/mobLinks.js:12]: Change href to `to`

**Review by ncutixavier — CHANGES_REQUESTED** [review]
image
Add your logo on mobile

File: src/components/mobLinks.js (hunk 2 above)
Comment (Code-250) [reply src/components/mobLinks.js]: just changed it

**Review by niyonkurueric — APPROVED** [review]

**Pull request merged**